www-apps/radarr/radarr-4.1.0.6175.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+SRC_URI="
+	amd64? ( https://github.com/Radarr/Radarr/releases/download/v${PV}/Radarr.master.${PV}.linux-core-x64.tar.gz )
+	arm? ( https://github.com/Radarr/Radarr/releases/download/v${PV}/Radarr.master.${PV}.linux-core-arm.tar.gz )
+	arm64? ( https://github.com/Radarr/Radarr/releases/download/v${PV}/Radarr.master.${PV}.linux-core-arm64.tar.gz )
+"
+
+DESCRIPTION="A fork of Sonarr to work with movies a la Couchpotato."
+HOMEPAGE="https://www.radarr.video"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="bindist strip test"
+
+RDEPEND="
+	acct-group/radarr
+	acct-user/radarr
+	media-video/mediainfo
+	dev-libs/icu
+	dev-util/lttng-ust:0
+	dev-db/sqlite"
+
+S="${WORKDIR}/Radarr"
+
+src_install() {
+	newconfd "${FILESDIR}/${PN}.conf" ${PN}
+	newinitd "${FILESDIR}/${PN}.init" ${PN}
+
+	keepdir /var/lib/${PN}
+	fowners -R ${PN}:${PN} /var/lib/${PN}
+
+	insinto /etc/${PN}
+	insopts -m0660 -o ${PN} -g ${PN}
+
+	insinto /etc/logrotate.d
+	insopts -m0644 -o root -g root
+	newins "${FILESDIR}/${PN}.logrotate" ${PN}
+
+	dodir  "/usr/share/${PN}"
+	cp -R "${S}/." "${D}/usr/share/radarr" || die "Install failed!"
```

Comment:
not doins?

Reply:
This approach is what's documented in the devmanual: https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html#trivial-installs

If you have an alternative that's better, can you please share the specifics?

Reply:
I'm not sure why the devmanual recommends that (I was just going to say `doins -r "${S}"/.' etc but if it says it, I assume it does for a reason), so nevermind!